### ISSUE TITLE
Picker in iOS 14

### ISSUE THREAD

**Issue by jminutaglio** [DDraiman1990/DroidTimeSelection#5]
Apple changed the default picker view in iOS14, which is no longer the 'spin wheel picker'.

Can this be a config option in DroidTimeSelection?  IE: select the older picker

**Comment by DDraiman1990** [comment]
Haven't had a lot of time to work on this due to juggling full-time job and freelancing but I will take a look this weekend and make it happen if I can spare the time.

**Comment by jminutaglio** [comment]
I can help in addition to just pointing out the issues..  :) 

```
if #available(iOS 13.4, *) {
            picker.preferredDatePickerStyle = .wheels
        } else {
            // Fallback on earlier versions
        }
```

TY for making this project, I love it and am looking forward to using it.

**Comment by DDraiman1990** [comment]
Fixed in the new refactor. Version 2.0.0
